omero/developers/Model/StructuredAnnotations.rst

```diff
@@ -7,9 +7,7 @@ annotations are designed for individualized use by both sites and tools.
 Annotations can be attached to multiple instances simultaneously to
 quickly annotated all entities in a view. Each annotation has a "name"
```

Comment:
- `annotated` → `annotate`.
- `Each annotation has a "name" which can be interpreted as a "namespace" by tools`. I thought each annotation had a namespace (possibly in addition to a name)? Worth clarifying?